### ISSUE TITLE
Archive this repo

### ISSUE THREAD

**Issue by jgsogo** [conan-io/cci-sources-backup#3]
This repository is being superseded by https://c3i.jfrog.io/ui/repos/tree/General/cci-sources-backup. We need to archive (and delete) the repo.

If removing, this PR needs to be merged first: https://github.com/conan-io/conan-center-index/pull/7685
